queue_job/models/queue_job.py

```diff
@@ -281,10 +281,18 @@ def autovacuum(self):
                 ['|',
                  ('date_done', '<=', deadline),
                  ('date_cancelled', '<=', deadline),
-                 ('channel', '=', channel.complete_name)],
+                 ('channel', '=', channel.complete_name)], limit=1000
```

Comment:
won't it be better to set it up in the channel parameter like for the removal interval?
Otherwise LGTM

Reply:
agree

Reply:
I'm not sure it's useful, 1000 is generally a good number for this kind of batch operations, do you see cases where we would change this number?

Reply:
I agree with @guewen, i would keep the 1000  because I don't see benefits to set that number as a parameter